### PR TITLE
meta.yaml update

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by alexlang74 — APPROVED** [review]

**Comment by cdeepali** [comment]
Need to update the License and README here.

**Comment by npanpaliya** [comment]
pybind11 2.8.1 is available with anaconda. I see the feedstock builds 2.9. Can we not use 2.8.1? I was thinking if we could avoid forking this into Open-CE.

**Comment by ritumd** [comment]
> pybind11 2.8.1 is available with anaconda. I see the feedstock builds 2.9. Can we not use 2.8.1? I was thinking if we could avoid forking this into Open-CE.

I will check that and update accordingly

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,14 +16,14 @@ source:
...
-  number: 0
+  number: 1
 
 outputs:
   - name: pybind11-abi
     version: {{ abi_version }}
     build:
       noarch: generic
-      skip: true  # [not linux64]
+      #skip: true  # [not linux64]
```
Comment (cdeepali) [inline recipe/meta.yaml]: We can remove this commented line.

**Comment by ritumd** [comment]
> pybind11 2.8.1 is available with anaconda. I see the feedstock builds 2.9. Can we not use 2.8.1? I was thinking if we could avoid forking this into Open-CE.

Hi Nishidha, Pybind11-abi is required to build mamba and its not available in anaconda 2.8.1.

**Review by cdeepali — APPROVED** [review]

**Review by cdeepali — APPROVED** [review]

**Pull request merged**